llvm/test/CodeGen/AMDGPU/scc_quadmask.ll

```diff
@@ -0,0 +1,57 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
```

Comment:
Suggest just adding tests to the end of `llvm.amdgcn.quadmask.ll` instead of creating a new file.

Reply:
yes

Reply:
Tests moved to `llvm.amdgcn.quadmask.ll`.  RUN lines modified.